language/stdlib/modules/libra_system_2.move

```diff
@@ -58,100 +57,111 @@ module LibraSystem2 {
     // It can only be called a single time. Currently, it is invoked in the genesis transaction.
     public fun initialize_validator_set() {
         Transaction::assert(Transaction::sender() == 0x1D8, 1);
-        move_to_sender<ValidatorSet>(ValidatorSet {
+
+        LibraConfig::publish_new_config<T>(T {
+            scheme: 0,
             validators: Vector::empty(),
-            last_reconfiguration_time: 0,
-            change_events: Event::new_event_handle<ValidatorSetChangeEvent>(),
         });
     }
 
+    // This returns a copy of the current validator set.
+    public fun get_validator_set(): T {
+        LibraConfig::get<T>(0x1D8)
+    }
+
+    // This copies the vector of validators into the LibraConfig's resource under ValidatorSet address
+    fun set_validator_set(value: T) {
+        LibraConfig::set<T>(0x1D8, value)
```

Comment:
not sure if this one-liner is worth the optimization

Reply:
but it does map to the get_validator_set above :O

Reply:
Right :) These two functions are from the Zekun's change: https://github.com/libra/libra/blob/c7bd7fc82233bdaecb93a7138e0520d2481b44ab/language/stdlib/modules/libra_system.move#L329